gnd/src/main/res/layout/map_layers_popup_window.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView
```

Comment:
Could you add some horizontal padding to the dialog (equal to the top and bottom padding)? I know there's a little more UI work to do on this, but it's an easy fix that might add a little polish in the meantime.